### PR TITLE
Create new no result page

### PR DESCRIPTION
**Pull request by NohYujeong**

### CODE REVIEW

File: app/components/articleSearch/components/requestPaperDialog/index.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+import validateEmail from "../../../../helpers/validateEmail";
+import ScinapseFormikInput from "../../../common/scinapseInput/scinapseFormikInput";
+import ReduxAutoSizeTextarea from "../../../common/autoSizeTextarea/reduxAutoSizeTextarea";
+import { ACTION_TYPES } from "../../../../actions/actionTypes";
+import ActionTicketManager from "../../../../helpers/actionTicketManager";
+import Icon from "../../../../icons";
+declare var ga: any;
+const styles = require("./requestPaperDialog.scss");
+
+const LAST_SUCCEEDED_EMAIL_KEY = "l_s_e_k";
```
Comment (TylorShin) [inline app/components/articleSearch/components/requestPaperDialog/index.tsx]: why you re-declare this constant again? it's in `fullTextDialog.tsx`. if you want to use this key in other places, just pull out the constant from `fullTextDialog.tsx` and make some `constant.tsx` in a proper position. then import & use it.

File: app/components/articleSearch/index.tsx [hunk 2]
```diff
@@ -227,9 +234,7 @@ class ArticleSearch extends React.PureComponent<ArticleSearchContainerProps, Art
             currentUser={currentUserState}
             papers={searchItemsToShow}
             isLoading={isContentLoading}
-            searchQueryText={
-              articleSearchState.searchFromSuggestion ? articleSearchState.suggestionKeyword : queryParams.query
-            }
+            searchQueryText={articleSearchState.suggestionKeyword || queryParams.query}
```
Comment (TylorShin) [inline app/components/articleSearch/index.tsx]: is it okay?

Comment (TylorShin) [reply app/components/articleSearch/index.tsx]: i mean `articleSearchState.suggestionKeyword` always exists?

File: app/components/articleSearch/index.tsx [hunk 3]
```diff
@@ -204,13 +204,20 @@ class ArticleSearch extends React.PureComponent<ArticleSearchContainerProps, Art
       );
     } else if (
       hasNoSearchResult &&
-      articleSearchState.matchAuthors &&
-      articleSearchState.matchAuthors.totalElements === 0 &&
-      queryParams
+      queryParams &&
+      (!articleSearchState.matchAuthors ||
+        (articleSearchState.matchAuthors && articleSearchState.matchAuthors.totalElements === 0))
```
Comment (TylorShin) [inline app/components/articleSearch/index.tsx]: it's too verbose. make it one variable like `noAuthorResult` or sth like that.

File: app/components/articleSearch/index.tsx (hunk 2 above)
Comment (NohYujeong) [reply app/components/articleSearch/index.tsx]: Yes it always exists. If there is no matching value, it goes down to "".

File: app/containers/paperShowActionBar/components/constant.tsx [hunk 4]
```diff
@@ -0,0 +1 @@
+export const LAST_SUCCEEDED_EMAIL_KEY = "l_s_e_k";
```
Comment (TylorShin) [inline app/containers/paperShowActionBar/components/constant.tsx]: i think it's not proper location to here.
maybe `app/common/constants` or `app/constants` will be better.

File: app/components/articleSearch/components/noResult/noResultContent.tsx [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+interface NoResultContentProps {
+  hasEmptyFilter: boolean;
+  doiPatternMatched: boolean;
+  doi: string | null;
+  searchText: string | null;
+  handleSetIsOpen: () => void;
+}
+
+function disabledFilterMessage(hasEmptyFilter: boolean) {
```
Comment (TylorShin) [inline app/components/articleSearch/components/noResult/noResultContent.tsx]: in this case, you can use `const DisabledFilterMessage: FunctionComponent<{hasEmptyFilter: boolean}> = () => React.memo{}` something like `pureComponent`.

**Pull request merged**